src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const store = {};
+
+  store.count = 0;
+
+  const funcObj = function(x) {
+    store.count++;
+
+    if (store.count >= 4 && store.count % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    }
+
+    return (y) => x + y;
```

Comment:
you're returning function in both conditions, just create an inner function that will return you one of the values and return it

Reply:
refactored

hope I've got your advises right

![image](https://user-images.githubusercontent.com/120763734/222856242-695273c0-4cd8-4d52-ab36-1a6f95d63b5f.png)